.travis.yml

```diff
@@ -69,39 +72,30 @@ env:
 # Install system dependencies, namely a very barebones ROS setup.
 before_install:
   - sudo sh -c "echo \"deb http://packages.ros.org/ros/ubuntu $ROS_CI_DESKTOP main\" > /etc/apt/sources.list.d/ros-latest.list"
-  - sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
-  - sudo apt-get update -qq
-  - sudo apt-get install -y python-catkin-pkg python-rosdep python-wstool ros-$ROS_DISTRO-catkin
+  - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+  - sudo rm /var/lib/dpkg/lock
```

Comment:
This should not be necessary in nominal conditions.

Reply:
Ok! I used the commands reported in the installation instructions of ROS Kinetic Kame, official web site.